src/formBuilder/utils.js

```diff
@@ -691,11 +691,18 @@ function generateSchemaElementFromElement(element: ElementProps) {
       element.schema !== undefined && element.schema.description !== undefined
         ? element.schema.description
         : element.dataOptions.description;
-    return {
+
+    const returnElement = {
       $ref: element.$ref,
       title: title,
       description: description,
     };
+
+    return element.schema !== undefined &&
+      element.schema.required !== undefined &&
+      element.schema.required.length !== 0
```

Comment:
Isn't required type of boolean. Meaning getting it length to evaluate its not zero is unnecessary. Rather evaluate if required is equal to `true`

Reply:
Thanks @cnxumalo. However, required is an array thus I check its length.

Reply:
Oh, I see